assignments/this.js

```diff
@@ -1,26 +1,59 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. it is using this while intentionally or unitetionally pointing to the window. 
+* 2. The implicit binding is reffering to the left of the dot and binds to that scope
+* 3. The  new  binding uses this while referencing a constructor to build a new object using the constructor.
+* 4. this is used in explicit binding to reference another constructor for inheritance .
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
-
+console.log(this);
 // code example for Window Binding
 
 // Principle 2
 
 // code example for Implicit Binding
```

Comment:
Hey Thomas, it looks like you copy and pasted work from the other half of this project for your examples. But they don't run for me. Technically, the examples are correct, but please write out individual examples each time on similar projects in the future.